core-infrastructure/src/db/Core.Database/Scripts/112-CreateTablesFileIntegrity.sql

```diff
@@ -0,0 +1,34 @@
+IF NOT EXISTS(SELECT *
+              FROM INFORMATION_SCHEMA.TABLES
+              WHERE table_name = 'DatasetType')
+    BEGIN
+        CREATE TABLE dbo.DatasetType
+        (
+              DatasetName varchar(16)   NOT NULL,
```

Comment:
I think I called this "short-code" in the original ticket; it'd be nice if everything had a decent initialism (a là AAR, CFR, etc.) but I'm not sure this holds true for all…?

Reply:
Will change this to `DatasetShortCode` as things do appear to have a concise, human-readable reference.